app/code/Magento/Cron/Console/Command/CronCommand.php

```diff
@@ -90,6 +103,10 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        if ($this->maintenanceMode->isOn()) {
+            $output->writeln('<info>' . 'Cron is disabled because the maintenance mode is enabled.' . '</info>');
+            return;
+        }
         if (!$this->deploymentConfig->get('cron/enabled', 1)) {
```

Comment:
Please pay attention to this configuration. You already may disable cron. Add cron/enabled as 0 to configuration file. There is no special CLI command for this setting for now.